### PR TITLE
Add twelveHour feature; visibility of colons

### PR DESCRIPTION
**Pull request by Offlein**
Enable 12-hour clock option; don't not-display colons if blinking, since this looks bad on some fonts.

### CODE REVIEW

**Comment by Offlein** [comment]
Sorry, almost forgot tests.

File: Clock.vue [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
   <time class="clock">
     <span class="clock__hour">{{ hours }}</span><!--
-    --><span v-if="!blink || seconds % 2 === 0">:</span><!--
-    --><span v-else>&nbsp;</span><!--
+    --><span class="clock__colon" v-bind:style="{ visibility: (!blink || seconds % 2 === 0) ? 'visible' : 'hidden' }">:</span><!--
     --><span class="clock__minutes">{{ minutes }}</span><!--
-    --><span v-if="displaySeconds && (!blink || seconds % 2 === 0)">:</span><!--
-    --><span v-else-if="displaySeconds">&nbsp;</span><!--
-    --><span v-if="displaySeconds" class="clock__seconds">{{ seconds }}</span>
+    --><span class="clock__colon" v-if="displaySeconds" v-bind:style="{ visibility: (!blink || seconds % 2 === 0) ? 'visible' : 'hidden' }">:</span><!--
```
Comment (frinyvonnick) [inline Clock.vue]: Maybe you could use the [shorthand](https://vuejs.org/v2/api/#v-bind) of `v-bind`? In this case `v-bind:style` becomes `:style` 😄

Comment (Offlein) [reply Clock.vue]: Yeah, OK I'm into that!

Comment (Offlein) [reply Clock.vue]: Hey wait, you're not even @eddyerburgh ... What do you, just review random pull requests?? 😆 Either way, I appreciate it. :)

Comment (frinyvonnick) [reply Clock.vue]: Sure, i contributed to the project. So i have notifications on pull requests and issues 👍

Comment (Offlein) [reply Clock.vue]: Oh, yeah. It says "Contributor" next to your name. I'm dumb!

**Comment by eddyerburgh** [comment]
Thank you :)

**Comment by Offlein** [comment]
Thank YOU!

**Comment by eddyerburgh** [comment]
Published in 2.0.0

**Comment by Offlein** [comment]
Eyyyy 🍰 🎈

**Pull request merged**